infrastructure/code/template/details.html

```diff
@@ -600,8 +947,9 @@ <h2 class="example-fixes-header">Fixed Code Examples</h2>
       };
 
       // Event listeners for main and nested tabs
+      // TODO: move this into the HTML as an onclick
```

Comment:
I think we can simplify a lot of this code by just adding an `onclick` attribute to these elements and in IntelliJ and VSCode we only implement the functions that these elements use. The changes are already complicated enough in this PR and its other two in our IDEs so I'd like to do that in a follow-up if there's time.

Reply:
The mark-up is 😍 , thank you! 
It seems that there are some issues with the listeners injected from the IDE? You can see that error when switching between issues.

<img src="https://github.com/snyk/snyk-ls/assets/1948377/e99367fd-28f1-45f6-b99e-56a30c2c98e2" width="460">

Reply:
How did you manage to test this view? The consistent ignores backend is broken at the moment (it was broken on Friday and I explained in the PR description and it seems still broken today), so I haven't been able to test this part of the UI (where we have an ignored issue)

Reply:
In dev, everything is working fine 🙌

Reply:
Also, did we say that is the issue is ignored, we don't show AI Fixes? I'm not sure now and I can't find references to this case.

Reply:
Based on the designs we're meant to show the `Fix Analysis` tab: https://www.figma.com/design/hcI2QHUtHfcIjgrpYlMqff/Holistic-Ignores?node-id=2292-64060&t=nsuVsneQwSDwzp71-4. I'll change it in this PR

Reply:
Thanks it does work in dev indeed! I've pushed a fix in https://github.com/snyk/vscode-extension/pull/467/commits/b21e4068514cf60caa82c48230e71b0c7c87c734